xbmc/Application.cpp

```diff
@@ -2356,6 +2356,15 @@ bool CApplication::OnAction(const CAction &action)
       g_windowManager.ActivateWindow(WINDOW_MUSIC_PLAYLIST);
     return true;
   }
+  if (action.GetID() == ACTION_PLAYER_PROCESS_INFO)
+  {
+    if ((m_pPlayer->IsPausedPlayback() || m_pPlayer->IsPlaying()) &&
```

Comment:
Confused: which one can be dropped? m_pPlayer->IsPausedPlayback() or m_pPlayer->IsPlaying() ? How to detect whether player is active but paused?